portfolio/src/main/webapp/style/style.css

```diff
@@ -24,20 +24,32 @@ a:hover, a:active {
 }
 
 h1 {
-    font-size: 70px;
+  width: 100%;
+  background-color: #34623F;
+  color: white;
+  font-size: 70px;
 }
 
 p {
   font-size: 20px;
 }
 
 ul {
-    display: flex;
-    align-content: flex-end;
-    margin-right: 50px;
+  display: flex;
+  align-content: flex-end;
+  margin-right: 50px;
 }
 
 #picDesc {
```

Comment:
nit: the [style guide](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters) suggests using hyphen to separate IDs, so "pic-desc" here and "band-container" below